model/riscv_insts_zfh.sail

```diff
@@ -0,0 +1,916 @@
+
```

Comment:
Should probably have the copyright header from fext given this is heavily derived from it, as well as an equivalent to:
```
/* **************************************************************** */
/* This file specifies the instructions in the F extension          */
/* (single precision floating point).                               */

/* RISC-V follows IEEE 754-2008 floating point arithmetic standard. */

/* Original version written by Rishiyur S. Nikhil, Sept-Oct 2019    */

/* **************************************************************** */
/* IMPORTANT!                                                       */
/* The files 'riscv_insts_fext.sail' and 'riscv_insts_dext.sail'    */
/* define the F and D extensions, respectively.                     */
/* The texts follow each other very closely; please try to maintain */
/* this correspondence as the files are maintained for bug-fixes,   */
/* improvements, and version updates.                               */

/* **************************************************************** */
```
That should probably also be updated across all three to reference Zfh

Reply:
(Normally wouldn't mandate that block of documentation but F and D, and now Zfh, are a bit special in how they're implemented and we want to ensure they stay as similar as possible)